.github/actions/set-env-variables/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
       run: |
         echo "QUAY_ORGANIZATION=cilium" >> $GITHUB_ENV
         echo "QUAY_ORGANIZATION_DEV=cilium" >> $GITHUB_ENV
-        # no prod yet
+        echo "QUAY_CHARTS_ORGANIZATION=cilium/charts" >> $GITHUB_ENV
         echo "QUAY_CHARTS_ORGANIZATION_DEV=cilium-charts-dev" >> $GITHUB_ENV
```

Comment:
Currently discussing with @aanm but I'm leaning towards consolidating these into:

- quay.io/cilium/charts/cilium
- quay.io/cilium/charts/cilium-ci

We can then deprecate the use of the `cilium-charts-dev` org (which I think was primarily useful just to establish the pattern in the first place).

Reply:
(For this PR you wouldn't need to update the cilium-charts-dev case, we can fix that up in the other workflow)